pkg/plugins/plugindef/plugindef.cue

```diff
@@ -31,9 +35,116 @@ seqs: [
 				// a superset of plugin types.
 				#IncludeType: type | "dashboard" | "page"
 
-				// Human-readable name of the plugin that is shown to the user in
-				// the UI.
-				name: string
+				// Metadata about the plugin.
```

Comment:
Style nit: In blocks below, periods not needed at line ends unless they close a complete sentence.